File: trunk/gFiveFinanzas/src/main/java/com/gfive/domain/rest/PedidoList.java
package com.gfive.domain.rest;

import java.util.List;
import javax.xml.bind.annotation.XmlRootElement;

	@XmlRootElement(name="pedidos")
	public class PedidoList {

		private List<PedidoRest> data;

		public List<PedidoRest> getData() {
			return data;
		}

		public void setData(List<PedidoRest> data) {
			this.data = data;
		}
	}

File: GFive/GF.Portal/GF.Layer.BL.BE/PedidoList.vb
Imports System.Xml.Serialization

<XmlRoot("pedidos")> _
Public Class PedidoList
    <XmlElement()> _
    Public data As List(Of PedidoRest)
    '<XmlArrayItem("data")> _
    'Public Property data() As List(Of PedidoRest)
    '    Get
    '        Return _data
    '    End Get
    '    Set(ByVal value As List(Of PedidoRest))
    '        _data = value
    '    End Set
    'End Property
End Class
